packages/tokamak/sdk/src/adapters/standard-bridge.ts

```diff
@@ -286,6 +286,9 @@ export class StandardBridgeAdapter implements IBridgeAdapter {
     l1Token: AddressLike,
     l2Token: AddressLike
   ): boolean => {
+    if (!this.messenger.nativeTokenAddress) {
```

Comment:
I think we can assume below.

1. not to set nativeTokenAddress, use l1Token as the nativeTokenAddress.
I think it will return err later (when user use getMessageStatus... etc)

Can you check it ?

Reply:
yes, this function will throw the error because it doesn't cast `undefined` to hex by using `hexStringEquals` function

Reply:
But I think this function will return true above assumption...
Is it right ?

Reply:
@rlgns98kr the function `filterOutL2NativeTokenDepositsAndWithdrawls` will return "true" and the function `filterL2NativeTokenDepositsAndWithdrawls`  will return "false" when the native token is undefined/empty.

We assume that if the native token is undefined, we will filter out this native token bridge(don't support).